CryptoPkg/Library/OpensslLib/OpensslLibAccel.inf

```diff
@@ -5,6 +5,7 @@
 #

```

Comment:
Please change the description too.

This module provides OpenSSL Library implementation with TLS features
along with performance optimized implementations of SHA1, SHA256, SHA512,
AESNI, VPAED, and GHASH for IA32 and X64 and **AARCH64**.